app/views/users/show.html.erb

```diff
@@ -1,6 +1,11 @@
 <div>
-  <h1>You're connected to the user show page!</h1> <%= button_to 'Add Partner', {:controller => "partners", :action => "create", :requester_id => current_user.id, :requestee_id=> @user.id} , {:method=>:post}  %>
-  <%= link_to "Not my partner anymore!", partner_path(@user), data: { turbo_method: :delete, turbo_confirm: "Are you sure?" }, class: "btn btn-primary" %>
+  <h1>You're connected to the user show page!</h1>
+  <% if @partner.nil? %>
+    <%= button_to 'Add Partner', {:controller => "partners", :action => "create", :requester_id => current_user.id, :requestee_id=> @user.id} , {:method=>:post}  %>
```

Comment:
We aren't able to use this to add our button classes unfortunately, I don't know if there's a way to modify a button_to directly, but button_to implicitly only takes 3 args. Either we'll have to use a link_to or create a css class specifically for button_to (if that's possible), otherwise it's pretty ugly. I know this was a workaround for how partners used to work, either way, maybe not a huge priority but just wanted to put it on our radar.

Reply:
Yeah, I'd prefer to have bootstrap classes. I think it's possible by adding it to the las hash:
https://stackoverflow.com/questions/14123884/rails-button-to-applying-css-class-to-button